### PR TITLE
Upgrading plugins.

### PR DESCRIPTION
**Pull request by sduskis**
Basically ran "mvn versions:display-plugin-updates" to get the latest versions.
Also removed some explicit version settings in child pom.xml and moved them to depend on the top pom.xml's versions.

### CODE REVIEW

File: bigtable-client-core-parent/bigtable-client-core/pom.xml [hunk 1]
```diff
@@ -133,7 +133,6 @@ limitations under the License.
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-release-plugin</artifactId>
-                <version>2.5.1</version>
```
Comment (rameshdharan) [inline bigtable-client-core-parent/bigtable-client-core/pom.xml]: How come the version was removed in this case? I guess that makes it default to 'latest' but still makes me a bit nervous, maybe we can pin it more explicitly?

File: bigtable-dataflow-parent/bigtable-dataflow-import/pom.xml [hunk 2]
```diff
@@ -124,7 +124,6 @@ limitations under the License.
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-eclipse-plugin</artifactId>
-                <version>2.9</version>
```
Comment (rameshdharan) [inline bigtable-dataflow-parent/bigtable-dataflow-import/pom.xml]: Same question here, how come no version for maven-eclipse-plugin?

**Comment by rameshdharan** [comment]
LGTM except curious why the versions were removed for two of the plugins.

File: bigtable-dataflow-parent/bigtable-dataflow-import/pom.xml (hunk 2 above)
Comment (sduskis) [inline bigtable-dataflow-parent/bigtable-dataflow-import/pom.xml]: It was removed from the child, and now the version is managed in the top pom.xml for all usages of maven-eclipse-plugin

File: bigtable-client-core-parent/bigtable-client-core/pom.xml (hunk 1 above)
Comment (sduskis) [inline bigtable-client-core-parent/bigtable-client-core/pom.xml]: This isn't latest... it depends on the top pom's version management.

**Comment by sduskis** [comment]
The top pom has dependencyManagement and pluginManagement which drive the version management for child poms.

**Comment by rameshdharan** [comment]
Got it, thanks for clarifying.

**Pull request merged**